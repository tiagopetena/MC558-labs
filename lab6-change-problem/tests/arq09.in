50 10000
143 883 3
119 200 1
134 330 2
93 107 1
8 231 1
5 397 3
99 209 1
39 285 2
15 131 3
94 525 1
76 991 1
70 671 3
127 213 1
10 177 1
33 471 2
116 767 2
126 159 3
31 965 3
121 251 3
90 515 3
78 564 1
107 970 2
108 683 2
77 934 2
44 576 3
106 755 3
26 365 3
129 619 2
136 402 3
73 235 2
110 715 2
87 97 1
54 853 1
37 225 2
141 297 3
38 9 2
35 841 3
117 473 2
32 872 2
7 173 2
86 873 1
64 757 1
97 249 3
56 337 3
92 906 3
59 980 3
118 783 1
20 173 2
58 547 2
147 373 1
