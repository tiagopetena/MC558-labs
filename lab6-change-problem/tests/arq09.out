7909 52955
